common/beerocks/btl/CMakeLists.txt

```diff
@@ -12,11 +12,11 @@ find_package(MapfTransport QUIET)
 
 if(MapfTransport_FOUND)
 message(STATUS "${BoldGreen}Using MultiAP Framework for BTL transport${ColourReset}")
-set(BTL_TYPE "LOCAL_BUS" CACHE STRING "" INTERNAL)
+set(BTL_TYPE "LOCAL_BUS" PARENT_SCOPE)
```

Comment:
So that *does* work in the end!

What is the effect exactly? Visibility in the btl/CMakeLists.txt, or up to the common/CMakeLists.txt?

Reply:
"If the PARENT_SCOPE option is given the variable will be set in the scope above the current scope."
So in this case, it will be visible in btl/CMakeLists.txt and any subdirectory (this is try for any SET) but also for the upper directory, which is indeed common/CMakeLists.txt